### PR TITLE
Fixed the issue where the balance could not be refreshed in time afte…

### PR DESCRIPTION
**Pull request by peterQin0826**
…r the transfer was completed

## What does this implement/fix? Explain your changes.
Fixed the issue where the balance could not be refreshed in time after the transfer was completed

## Does this close any currently open issues?  
If it fixes a bug or resolves a feature request, be sure to link to that issue.
OneKeyHQ/TaskHub#582
## Pull request type
 
_Put an `x` in the boxes that apply_
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Where has this been tested?
…

## Any other comments?
…

### CODE REVIEW

File: android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/process/SendHdActivity.java [hunk 1]
```diff
@@ -869,6 +867,15 @@ private void keyBoardHideRefresh() {
     protected void onResume() {
         super.onResume();
         isResume = true;
+        String walletName = new AccountManager(mContext).getCurrentWalletName();
```
Comment (QuincySx) [inline android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/process/SendHdActivity.java:870]:
```
// 直接声明
private AppWalletViewModel mAppWalletViewModel;

public void initView() {
        mAppWalletViewModel = getApplicationViewModel(AppWalletViewModel.class);
        mAppWalletViewModel.currentWalletBalance.observe(this, balance -> {
            // 可以直接观察金额变化
        });
}
```

Comment (peterQin0826) [reply android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/process/SendHdActivity.java:870]: 可以，但这样会一直接收事件

Comment (QuincySx) [reply android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/process/SendHdActivity.java:870]: 一直实时接收金额，逻辑也没什么太大问题。
如果影响业务不想一直接收也没啥大问题，new BalanceManager() 
 new AccountManager() 要拿出去，就不要每次 onResume 都 new 一个了。

Comment (peterQin0826) [reply android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/process/SendHdActivity.java:870]: 对，考虑到还是需要实时刷新当前余额，体现真实余额

Comment (QuincySx) [reply android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/process/SendHdActivity.java:870]: 稍后我也在 ViewModel 中加个过滤，如果金额没变化，就不发通知了。

**Review by QuincySx — APPROVED** [review]

**Pull request merged**